lib/command.c

```diff
@@ -1615,8 +1615,11 @@ permute (struct graph_node *start, struct vty *vty)
       bool skip = false;
       if (stok->type == FORK_TKN && tok->type != FORK_TKN)
         for (ALL_LIST_ELEMENTS_RO (position, ln, gnn))
-           if (gnn == gn && (skip = true))
```

Comment:
Yes, I know that's not a bug, we should set the skip to true.
I just remove the assignment operation (skip=true) from the condition statement to make the coverity scan won't issue **PW.ASSIGN_WHERE_COMPARE_MEANT** again.
Or maybe we can tell the covertiy scan to ignore this warning ?